package.json

```diff
@@ -10,29 +10,32 @@
   "private": true,
   "devDependencies": {
     "@agrc/eslint-config": "^1.3.0",
+    "@arcgis/core": "^4.18.1",
     "@babel/cli": "^7.12.10",
     "@babel/core": "^7.12.10",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-proposal-optional-chaining": "^7.12.7",
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
-    "@storybook/addon-essentials": "^6.1.14",
-    "@storybook/addon-storyshots": "^6.1.14",
-    "@storybook/addons": "^6.1.14",
-    "@storybook/react": "^6.1.14",
+    "@storybook/addon-essentials": "^6.2.3",
+    "@storybook/addon-storyshots": "^6.2.3",
+    "@storybook/addons": "^6.2.3",
+    "@storybook/react": "^6.2.3",
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-require-context-hook": "^1.0.0",
     "babel-plugin-transform-rename-import": "^2.3.0",
     "chokidar-cli": "^2.1.0",
-    "copy-webpack-plugin": "^7.0.0",
+    "copy-webpack-plugin": "^6.*",
     "copyfiles": "^2.4.1",
     "core-js": "^3.8.2",
     "eslint": "^7.17.0",
     "esri-loader": "^3.0.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.6.3",
     "lerna": "^3.22.1",
+    "react": "*",
+    "react-dom": "^17.0.2",
```

Comment:
**question**: will you explain this one to me

Reply:
I was having troubles getting storybook to run. Let me take a look and see if this is still needed...

Reply:
I removed these...